docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting_discretechannels_[3.12.1,).partial.md

```diff
@@ -0,0 +1,5 @@
+## Retention policies with Channels
+{#RetentionpolicyTentaclecleanupandtroubleshooting-Retentionpolicywithchannels}
+[Channels](/docs/key-concepts/projects/channels.md) can be used in Octopus to handle many different deployment scenarioes. In some cases you may have a hotfix channel in which deployments, as they are promoted through their environments, should be considered as overriding deployments from the default channel for the given enviroment. Alternatively you may be using channels to deploy feature branches which may involve having several concurrent releases active at any one time across different channels for the same environment. When using the feature branch type scenario, you will likely want retention policies to factor in the channel when deciding which previous deployment files should be removed from your Tentacle. This behaviour can be enabled for each project via the `Discrete Channel Release` flag at under `Deployment Target settings` on the {{Project,Process}} page.
```

Comment:
* typo in 'scenarioes'.
* Needs a bit of re-wording - hard to grok it.
* wording for `Discrete Channel Release` is different to screenshot